src/z80asm/t/issue_341.t

```diff
@@ -40,18 +40,14 @@ my $map = join("\n", grep {/test.c:|test1.asm:/} split('\n', slurp("test.map")))
 check_text($map, <<'END', "map file contents");
 __C_LINE_0_test_2ec             = $0000 ; addr, local, , test_c, , test.c:0
 __C_LINE_1_test_2ec             = $0000 ; addr, local, , test_c, , test.c:1
-__C_LINE_2_test_2ec             = $0000 ; addr, local, , test_c, , test.c:2
```

Comment:
Are you sure removing these was an okay behavior? Why these lines no longer appear? A they empty?